readme.md

```diff
@@ -8,32 +8,80 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-**rehype-dom** is a DOM-based version of [rehype][] built on [unified][]!
+This project is a monorepo that contains alternatives to [rehype][] for use in
+browsers.
 
-*   [`rehype-dom`][api] — Programmatic interface
-*   [`rehype-dom-parse`][parse] — Parser
-*   [`rehype-dom-stringify`][stringify] — Stringify
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Example](#example)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This monorepo contains the following packages:
+
+*   [`rehype-dom-parse`][rehype-dom-parse]
+    — plugin to take HTML as input and turn it into a syntax tree (hast) in
+    browsers
+*   [`rehype-dom-stringify`][rehype-dom-stringify]
+    — plugin to take a syntax tree (hast) and turn it into HTML as output in
+    browsers
+*   [`rehype-dom`][api]
+    — `unified`, `rehype-dom-parse`, and `rehype-dom-stringify`, useful when
+    input and output are HTML
+
+## When should I use this?
+
+You can use this project when you want to use rehype in browsers.
+It’s much smaller compared to the APIs used by “normal” rehype because it uses
+DOM APIs to parse HTML and serialize hast.
```

Comment:
"normal" feels off here.
and the sentence as a whole feels a bit stilted.

```suggestion
It’s has a smaller bundle size, compared to the node focused rehype package, because rehype-dom uses
the browser DOM APIs to parse HTML and serialize hast.
```

Reply:
- There are a few grammatical things that look weird to me in your suggestion (`It’s has`, `node`, and `the browser DOM APIs`), perhaps when it is reworded it would make more sense to me?
- I don’t think it’s good to use `node focussed` for `rehype`. There are several good reasons to use `rehype` in the browser as detailed in the previous section. Highlighting that quality gives off the impressions that `rehype` does not work in browsers or should not be used there.

How about:

```
It has a smaller footprint in browsers compared to `rehype` itself as it uses
DOM APIs to do its work.
```

Reply:
Updated suggestion for more clarity.

> Highlighting that quality gives off the impressions that rehype does not work in browsers or should not be used there.

It can be run in browser, the circumstances where it should seem fairly limited.

I'd also be fine with your:

```
It has a smaller footprint in browsers compared to `rehype` itself as it uses
DOM APIs to do its work.
```

suggestion.

Reply:
> the circumstances where it should seem fairly limited.

I believe the main things that unified provides with ASTs, in comparison to a DOM, is positional info. The exact thing that this project cannot do. I would instead classify this projects use cases as being “fairly limited”.

---

Perhaps another good thing to add to the list, in a similar fain to positional info being of some importance, is control of how to format HTML: `rehype-stringify` *can* do it, but `rehype-dom-stringify` *cannot*.